internal/store/pod.go

```diff
@@ -1331,7 +1331,7 @@ func createPodStatusContainerReadyTimeFamilyGenerator() generator.FamilyGenerato
 			ms := []*metric.Metric{}
 
 			for _, c := range p.Status.Conditions {
-				if c.Type == v1.ContainersReady {
+				if c.Type == v1.ContainersReady && c.Status == v1.ConditionTrue {
```

Comment:
since the value is c.LastTransitionTime, I wonder if it would make sense to have a dimension "ready=True|False" instead? That way you can also track when a transition to not ready happened.

Reply:
The pattern I saw in other pod metrics was to have a [separate metric for the false case](https://github.com/kubernetes/kube-state-metrics/blob/main/internal/store/pod.go#L1543).

Reply:
Good point, since the metric is still experimental we can look at it later as well and work out things.